packages/edit-site/src/components/style-book/index.js

```diff
@@ -66,6 +69,8 @@ const STYLE_BOOK_IFRAME_STYLES = `
 		padding: 16px;
 		width: 100%;
 		box-sizing: border-box;
+		scroll-margin-top: var(--wp--style--root--padding-top);
+		scroll-margin-bottom: var(--wp--style--root--padding-bottom);
```

Comment:
These CSS variables aren't always present (they'll only be output if `useRootPaddingAwareAlignments` is `true`), and then it looks like the root padding winds up being overwritten by the `padding: 32px !important;` style for `body` further down in this file.

While hacking around locally, it seems to be working a bit better if we hard-code these values to `32px` instead:

```suggestion
		scroll-margin-top: 32px;
		scroll-margin-bottom: 32px;
```

Here's how it looks with hard-coded `32px`:

https://github.com/WordPress/gutenberg/assets/14988353/8bd74463-5ffc-4525-83af-4b29c0c02aa9

Whereas before it was still getting a bit cut off:

https://github.com/WordPress/gutenberg/assets/14988353/22f2e0c0-7eb9-4a11-ada5-d2029b828200

Reply:
fea8b96